webrtc.html

```diff
@@ -2251,6 +2251,10 @@
             state. Note the local or remote SDP descriptions in the
 	    previous stable state could be null if there has
             not yet been a successful offer-answer negotiation.</p>
+
+            <p>If the value of a <code>RTCRtpTransceiver.mid</code> was set to a non-null value by
+              an SDP description that is rolled back, the <code>RTCRtpTransceiver.mid</code> will be
+              set to null, as defined by [[!JSEP]].</p>
```

Comment:
OK, I added it to them.  Two of them had a clear ID.  The other.... I just made one up: "rollbacks".  Is that OK?